lib/node_modules/@stdlib/stats/incr/prod/lib/main.js

```diff
@@ -96,7 +96,7 @@ function incrprod() {
 
 		// Ensure fraction remains normalized to avoid overflow/underflow...
 		if ( frac > -0.5 && frac < 0.5 ) {
-			frexp( PARTS, frac );
+			frexp.assign( frac, PARTS );
```

Comment:
```suggestion
			frexp.assign( frac, PARTS, 1, 0 );
```

Missing stride and offset.